src/ontology/cl-edit.owl

```diff
@@ -32378,6 +32381,30 @@ SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000101
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002203 obo:CL_0000127))
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001307))
 
+# Class: obo:CL_4042023 (pthlh-expressing interneuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30134177") Annotation(oboInOwl:hasDbXref "PMID:37292997") obo:IAO_0000115 obo:CL_4042023 "A GABAergic interneuron that has its soma in a striatum. This GABAergic interneuron is characterised by the expression of PTHLH and PVALB.")
+AnnotationAssertion(terms:contributor obo:CL_4042023 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042023 "2024-09-17T10:20:15Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292997") oboInOwl:hasRelatedSynonym obo:CL_4042023 "PTHLH/PVALB cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292997") oboInOwl:hasRelatedSynonym obo:CL_4042023 "PTHLH +/PVALB+ interneuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292997") oboInOwl:hasRelatedSynonym obo:CL_4042023 "pvalb-expressing cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292997") rdfs:comment obo:CL_4042023 "According to spatial transcriptomic data in the human striatum, pthlh-expressing interneurons are one of the most abundant population of interneurons in the striatum (Leonardo D. Garma, et al., 2024).")
+AnnotationAssertion(rdfs:label obo:CL_4042023 "pthlh-expressing interneuron"@en)
+SubClassOf(obo:CL_4042023 obo:CL_0011005)
+SubClassOf(obo:CL_4042023 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0002435))
+SubClassOf(obo:CL_4042023 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000013433))
+SubClassOf(obo:CL_4042023 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000013502))
+
+# Class: obo:CL_4042024 (PTHLH MOXD1 expressing interneuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35308118") obo:IAO_0000115 obo:CL_4042024 "A pthlh-expressing interneuron that expresses MOXD1.")
```

Comment:
Wrong ref? Looks like different neuron types covered. Don't seem pthlh or MOXD1 in this paper